subsys/bluetooth/controller/ll_sw/ull_adv_sync.c

```diff
@@ -885,7 +884,7 @@ uint8_t ull_adv_sync_pdu_set_clear(struct lll_adv_sync *lll_sync,
 				   void *hdr_data)
 {
 	struct pdu_adv_com_ext_adv *ter_com_hdr, *ter_com_hdr_prev;
-	struct pdu_adv_ext_hdr *ter_hdr, ter_hdr_prev;
+	struct pdu_adv_ext_hdr ter_hdr = { 0 }, ter_hdr_prev = { 0 };
```

Comment:
iirc, structs can only be initialized when declared, so if we remove the initialization here, we'd need to a `memset` later or set the individual fields to 0.

Reply:
Change to `memset` increases FLASH size by 16 bytes in comparison to current version. Unfortunately I haven't got J-Trace to measure it during execution. I'm leaving it as it is implemented to avoid increase of FLASH size.

Reply:
The header is 1 bytes, use `memset` and compiler will optimize it to `*((uint8_t *)ter_hdr_prev) = 0U;`